### ISSUE TITLE
Suport to multiple applications is not work

### ISSUE THREAD

**Issue by netstart** [mgechev/angular2-seed#1434]
The problem occur when I run

```
 npm run serve.prod -- --app foo

or

 npm run build.prod -- --app foo
```

In the seeed.onfig.ts we have defined app,  statically.

![image](https://cloud.githubusercontent.com/assets/200232/19152879/7a056336-8baa-11e6-9c18-8d52fdd6a76b.png)

I believe, is better  use BOOTSTRAP_DIR variable

![image](https://cloud.githubusercontent.com/assets/200232/19152958/d4c9a598-8baa-11e6-925f-74404645b181.png)

**Comment by mgechev** [comment]
Officially is not stated that the seed support multiple applications, although people has tried to do that successfully in the [past](https://github.com/mgechev/angular2-seed/issues/1407).

Since it is not an official feature I'd recommend you to ask on gitter how to approach.

**Comment by netstart** [comment]
what? Not official feature?
But, in the readme here are instructions about it!

![image](https://cloud.githubusercontent.com/assets/200232/19168255/b15b8aaa-8be5-11e6-9ae3-9b8f9b6581e0.png)

**Comment by netstart** [comment]
I supose you need modify readme and remove this instructions

**Comment by mgechev** [comment]
Yes, you're correct. Just tried the seed with a couple of apps and it seems to work well. I'll update the configuration file and `build.js.ENV` in order to allow such scenarios.

**Comment by mgechev** [comment]
@netstart here's fix https://github.com/mgechev/angular2-seed/pull/1438.

**Comment by LiveLikeLastDay** [comment]
Is not working again, it was working back to "Minko Gechev b2a4173 docs: add basic info about prod build Fix #1738" on 1/1/2017.

After I pull the latest code from github today, the follow command only started the default "app"

`npm start -- --app my-app-name`
